RevenueCatUI/Views/RemoteImage.swift

```diff
@@ -17,44 +17,76 @@ import SwiftUI
 struct RemoteImage: View {
 
     let url: URL
+    let lowResUrl: URL?
     let aspectRatio: CGFloat?
     let maxWidth: CGFloat?
 
+    var fetchLowRes: Bool {
```

Comment:
This could be private

Reply:
Actually this is defining an (unnecessary IMO) duplicate source of truth.

Reply:
Hi Nacho! Thanks for reviewing the PR! I'm sorry we never got to work together!

As implemented, I didn't consider this a duplicate source of truth because it's a computed property, and this property makes other code more readable because the semantic meaning is clearly expressed rather than being derived inline whenever needed. Thoughts?

It could be private, and so could the four properties that precede it. I don't really have a strong opinion here but would be interested to hear any further thoughts you might have.